### DISCUSSION TITLE
Tasmota wlan usage over 24h

### DISCUSSION THREAD

**Discussion by Gifford47** [arendst/Tasmota#13886] [Q&A] [answered]
Hi!
I have several tasmota devices in use. many of them are tasmota s20 devices that are only switched on or off (about 4 times a day). I have noticed for some time that the same devices often send very different amounts in the wifi.
how can it be that one s20 socket consumes ~1MB/24h and another with exactly the same configuration and switching times ~8mb/24h?

i have flashed a completely new esp8266 with tasmota 10.0 and observed. it is only connected to the wifi, no mqtt or similar. nevertheless it also consumes approx.7-8 MB/24h.
my question now is how I can reduce the wifi consumption (eg sockets in the bedroom)?
is there perhaps a periodic send interval or keepalives that i could increase or similar?

![wlan_usage_24h](https://user-images.githubusercontent.com/49484063/144291815-d21874d4-5463-411c-848a-52876e2d587d.PNG)

**Comment by barbudor** [comment, marked as answer]
If you are not using MQTT then there is no periodic publication. Otherwise Teleperiod command can help to reduce the periodicity from the default 5min to longer or even disable.

You are not saying to use syslig so I would assume nothing here

Do you have any software that could be polling the device through http like the infamous TasmoAdmin? Or simply a PC that have a browser continuously open on the web page?

The best to analyse what is happening would be to use a network analysis tool somewhere on your network may be on your router itself

**Comment by Gifford47** [comment]
@barbudor thanks for your fast answer! i have no special software or a browser that connects to the node.
that was a good tip with the analysis tool.
i'm recording the traffic to the node right now with wireshark. as soon as i find something, i'll let you know.

**Reply by barbudor** [reply]
How are you recording with wireshark? 
If running that on a PC, it can only see the traffic between the device and itself.
As tasmota is wifi based, you need to be on the router, or use an dedicated AP and an ethernet tap to capture the traffic

**Reply by Gifford47** [reply]
Yes, I have a unifi ap that is connected to the tasmota device.
I am trying to catch the traffic direct from the ap but I seems that I have not the right command yet...

**Reply by Gifford47** [reply]
ok, i have now a 24h overview and can say that only the ARP and ntp protocol are sending data in and out. but only up to 700 bytes per request every hour. so i never come up to 8 MB/24h.
now i am investigating with ubiquiti if there is issue on their side ...
